fixtures/room/create_room_fixtures.py

```diff
@@ -164,7 +164,7 @@
     mutation {
         createRoom(
             name: "Kigali", roomType: "Meeting", capacity: 6, floorId: 4, officeId: 1,
-            calendarId:"andela.com_38363233383232303439@resource.calendar.google.com",
+            calendarId:"232k",
```

Comment:
Why was this changed?

Reply:
I would suggest not to round off the percentage values, let the front end team decide on the number of decimal places they'd like to render.

Reply:
The test that used this fixture was hitting the check for existing calender_ids and stopping at that point, returning the error raised by that check.
This change ensures it moves past it and does what is required by the test.